lib/Dialect/EmitC/Conversion/MHLOToEmitC.cpp

```diff
@@ -608,7 +621,7 @@ struct ConvertMhloToEmitCPass
     // MHLO region ops
     target.addIllegalOp<mhlo::ReduceOp,
                         mhlo::ReturnOp>();
-    // other MHLO ops
+    // Other MHLO ops
```

Comment:
```suggestion
    // Other MHLO ops.
```

Reply:
Well that is what we have discussed this morning and therefore I didn't adjust all of these *short* comments.

Reply:
Yeah I know we didn't found a general solution to this, but I suggested the change because you ended all the other // MHLO ops with a dot as well.